tutorial_src/lea/doc/leazad-en.tex

```diff
@@ -0,0 +1,57 @@
+\documentclass{spiral}
+  \def\title{Leaves}
+  \def\id{lea}
+  \def\contest{Example task}
+  \def\desc{}
+  \def\TL{1 s}
+  \def\ML{64 MB}
+
+\begin{document}
+    \makeheader
+
+    A tree is an undirected unweighted graph,
+    consisting of $n$ vertices and $n-1$ edges.
+    The characteristic feature of the tree is that it is connected and acyclic.
+    A leaf in a tree is a vertex that has it's degree equal to 1
+    (that is, only one edge is adjacent to it).
+    If a rooted tree is given, it means that a certain designated vertex is a root.
+    It is assumed that in a rooted tree the root cannot be a leaf.
+    You are given a $n$ - vertex rooted tree.
+    The root is the vertex with index 1.
+    Your task is to calculate the number of it's leaves.
+
+\section{Input}
+
+    A single integer $n$ ($n \leq 2 \cdot 10^5$) denoting the size of the tree.
```

Comment:
If the root cannot be a leaf then n cannot be 2.

Reply:
It is not called a leaf, but it can have degree 1. Fixed